EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -925,6 +926,22 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		072599A726A0CDC9007EC229 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "cd \"$BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH/\"\nchmod u+w *.dylib # required for xcodebuild\nfor libname in *.dylib\ndo\n    [ -e \"$libname\" ] || continue\n    install_name_tool -id \"@rpath/$libname\" $libname\n    codesign -f -s - $libname\ndone";
```

Comment:
Can we move this to a script file in utils? It's fine if not, I just think it will be easier to maintain when it can be viewed as not a single string.